### ISSUE TITLE
Support list of exceptions

### ISSUE THREAD

**Issue by wooorm** [gretzky/comparative-superlative#2]
Maybe it’s a good idea to support a list of exceptions, such as for words that should use `more` or `most`:

- [manic](https://en.wiktionary.org/wiki/manic)
- [batshit](https://en.wiktionary.org/wiki/batshit)
- ...

And maybe a list of uncomparables:

- [cripple](https://en.wiktionary.org/wiki/cripple)

**Comment by gretzky** [comment]
Good idea! it's been awhile since I've looked at this little script, I'll take a look again.
